packages/worker/src/workflow/threaded-vm.ts

```diff
@@ -225,7 +225,14 @@ export class VMWorkflowThreadProxy implements Workflow {
       throw new TypeError(`Got invalid response output from Workflow Worker thread ${output}`);
     }
     output.calls.forEach((call) => {
-      call.workflowInfo.unsafe.now = Date.now;
+      const info = call.workflowInfo;
```

Comment:
Why does this matter?

Reply:
Not sure what you mean. Are you asking why I care about rehydrating `unsafe.now`?

Reply:
I mean the cloning

Reply:
Oh, that... Just because `unsafe.now` is now readonly, so technically can't be modified in-place. But fair point, I'll cast the field to not readonly.